CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Code of Conduct
+
+We are committed to creating a welcoming and inclusive environment for all contributors.
+As such, we have adopted the following code of conduct, which all contributors are expected to follow:
+
+- Be respectful of others. Harassment or discrimination of any kind will not be tolerated.
+- Be mindful of your language and behavior when communicating with others.
+- Be open-minded and willing to learn from others.
+
+# Contributing Guidelines
+
+We welcome contributions to this repository. Here are some guidelines to follow when contributing:
+
+- <details><summary>Create a new branch for your changes in form <code>ab/xxx-feature-name</code></summary>
+
+  #### Where:
+    * `ab` - your initials, short name or something else that could help identify the author of the branch.
+      For example, the John Doe could use `jd`, `jhn`, `doe` etc.
+    * `xxx` - number of the issue your branch is solving. Optional.
+    * `feature-name` - some descriptive name for your branch.
+
+  Examples: `jd/add-contibuting-md`, `doe/1337-add-elite-user`.
+
+  #### Why
+  Contributors' initials helps to track the work. Multiple devs could work on the same issue and the same feature.
+  Also, it helps with determining owners of abandoned branches and ask them to clean them up.
```

Comment:
I guess this does not relate to our OSS projects, I think we should remove it, I don't really care about branch naming in contributor's fork

Reply:
removed